### DISCUSSION TITLE
[Question]: Understanding the GHCR image tags - cache? And alias for latest dev image?

### DISCUSSION THREAD

**Discussion by victorhooi** [blakeblackshear/frigate#12137] [Ask A Question]
### What is your question:

So per here:

https://docs.frigate.video/frigate/installation/#docker

There are GHCR (GitHub container registry) tags like so:
* stable - Standard Frigate build for amd64 & RPi Optimized Frigate build for arm64
* stable-standard-arm64 - Standard Frigate build for arm64
* stable-tensorrt

If you go to the Package homepage:

https://github.com/blakeblackshear/frigate/pkgs/container/frigate

you can see all the tags - e.g. `dev-88046eb`, or `cache-amd64`.

I'm just curious - what does the `cache` here stand for, or mean?

And is there some kind of alias tag for the latest dev build?

**Comment by NickM-27** [comment]
Cache is exactly that, an image used as a cache, not for users

There is no latest dev tag, this is intentional
